packages/extractor-vue/src/vue-extractor.ts

```diff
@@ -0,0 +1,64 @@
+import { parse, compileTemplate, SFCBlock } from "@vue/compiler-sfc"
+// @ts-expect-error temporary typing issue
+import babel from "@lingui/cli/api/extractors/babel"
```

Comment:
add `"@lingui/cli/api/extractors/babel": ["./packages/cli/src/api/extractors/babel"],` to root paths in `tsconfig.json`